packages/react-dom/src/client/ToStringValue.js

```diff
@@ -67,15 +51,27 @@ export opaque type TrustedValue: {toString(): string, valueOf(): string} = {
  *
  * If application uses Trusted Types we don't stringify trusted values, but preserve them as objects.
  */
-export function toStringOrTrustedType(value: any): string | TrustedValue {
-  if (
-    enableTrustedTypesIntegration &&
-    // fast-path string values as it's most frequent usage of the function
-    typeof value !== 'string' &&
-    isTrustedTypesValue(value)
-  ) {
-    return value;
-  } else {
-    return '' + value;
-  }
+export let toStringOrTrustedType: any => string | TrustedValue = toString;
+if (enableTrustedTypesIntegration && typeof trustedTypes !== 'undefined') {
```

Comment:
This seems like it would make this function not inlinable if the flag is on or dynamic which isn’t great because it used to be a simple `’’ +` that gets inlined.

Reply:
That’s actually got me looking at it in the first place — it wasn’t getting inlined with a dynamic flag in FB bundle. Lemme think about this more.

Reply:
If the flag is _on_, it's already not being inlined, even before this PR. Probably because the body is bigger. Since we're doing a function call either way, I think a *smaller* function (as in this PR) would be better than a larger one. It's also a single call (in this PR) rather than two calls (as in before).

Reply:
I'm going to get this in because it's not being inlined either way with the flag on. If we want to fix this, we'll have to inline it manually at concrete callsites.

Reply:
Why do we toString in user space instead of letting the browser do it again?

Seems like we only need it in some IE legacy thing and/or for the `javascript:` URL check so we should only ever do this at all in those cases.

Reply:
It's for IE9 I think? We could feature-test it I guess.